cou/steps/analyze.py

```diff
@@ -100,7 +127,12 @@ def __str__(self) -> str:
         :return: String representation of Application objects.
         :rtype: str
         """
-        return os.linesep.join([str(app) for app in self.apps])
+        return (
+            "Control Plane:\n"
+            + "\n".join([str(app) for app in self.apps_control_plane])
+            + "Data Plane:\n"
+            + "\n".join([str(app) for app in self.apps_data_plane])
+        )
```

Comment:
[non-blocking] this starts being a bit too complex for a single return. Perhaps it'd be better to construct the app lists in two variables and return a multiline f-string here.

Reply:
Currently it seem more expressive to me. If it even gets complicated we can split it.